atlas/AtlasMl/atlasml/dependencies.py

```
@@ -0,0 +1,37 @@
+
+    if not authorization_header:
+        logger.warning("No Authorization header provided")
+        raise RequiresAuthenticationException
+
+    return authorization_header
+
+
+class TokenValidator:
+    async def __call__(self, api_key: str = Depends(_get_api_key)) -> APIKeyConfig:
+        logger.info(f"Validating API key: {api_key}")
+        logger.info(f"Available API keys in settings: {[key for key in settings.get_api_keys()]}")
```

Ruff (0.11.9)

Line too long (98 > 88)

(E501)